src/views/product-landing/components/product-landing-blocks/index.tsx

```diff
@@ -14,6 +15,18 @@ function ProductLandingBlocks({ blocks }: ProductLandingBlocksProps) {
 			{blocks.map((block: ProductLandingBlock, idx: number) => {
 				const { type } = block
 				switch (type) {
+					case 'callout':
+						return (
+							<div key={idx} className={s.cardsMargin}>
```

Comment:
Thoughts on how to best handle margin since these cards aren't paired with a heading? See the description for my full thoughts.

Reply:
Maybe we'd have a new `calloutCardMargin` class here? Based on the design spec, looks like that could be something like:

```css
.calloutCardMargin {
  margin-top: 48px 0;
}
```